apps/web/src/components/ExperienceTreeView/ExperienceTreeView.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import { notEmpty } from "@gc-digital-talent/helpers";
+import { getLocale } from "@gc-digital-talent/i18n";
+import { Accordion, Alert, Button, TreeView } from "@gc-digital-talent/ui";
+import { useIntl } from "react-intl";
+import { Skill } from "~/api/generated";
+import ExperienceAccordion from "../UserProfile/ExperienceAccordion/ExperienceAccordion";
+
+interface ExperienceTreeViewProps {
+  skill: Skill;
+}
+
+const ExperienceTreeView = ({ skill }: ExperienceTreeViewProps) => {
+  const intl = useIntl();
+  const locale = getLocale(intl);
+  const experiences = skill.experiences
+    ? skill.experiences.filter(notEmpty)
+    : [];
+  return (
+    <TreeView.Root
+      title={skill.name[locale] ?? ""}
+      subtitle={skill.description ? skill.description[locale] : ""}
+    >
+      {experiences.length === 0 && (
+        <TreeView.Item>
+          <Alert.Root type="warning" data-h2-margin="base(0, 0)">
+            <Alert.Title>
+              {intl.formatMessage({
+                defaultMessage:
+                  "This required skill must have at least 1 résumé experience associated with it.",
+                id: "7XSyhV",
+                description: "Experience tree view error message",
+              })}
+            </Alert.Title>
+          </Alert.Root>
+        </TreeView.Item>
+      )}
+      {experiences.length > 0 &&
+        experiences.map((experience) => (
+          <TreeView.Item key={experience.id}>
+            <Accordion.Root
+              type="single"
+              collapsible
+              data-h2-margin="base(0, 0)"
+            >
```

Comment:
Ugh, sorry but I just noticed this. We should avoid adding margin since it will overwrite required styles, in this case it is causing problems with the margin making the border show a gap. You could wrap this in a `div` with negative margin to fix that issue.

```tsx
<div data-h2-margin="base(-x.5, 0)">
  <Accordion />
</div>
```
![Screenshot 2023-04-11 153600](https://user-images.githubusercontent.com/4127998/231270224-e3db9425-d943-415d-8b47-18e63a9458a6.png)

Reply:
Thanks for that! I fixed it in [bebc221](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6173/commits/bebc22173522ed79d6fb405a6a90d58aa28ca170)